modules/_labs/core/react/stories/stories.tsx

```diff
@@ -1,91 +1,84 @@
 /// <reference path="../../../../../typings.d.ts" />
 /** @jsx jsx */
 import {jsx} from '@emotion/core';
-import {storiesOf} from '@storybook/react';
 import styled from '@emotion/styled';
 import withReadme from 'storybook-readme/with-readme';
 import {spacing} from '@workday/canvas-kit-react-core';
 import type, {space} from '..';
 
 import README from '../README.md';
 
-export const inverseStyle = {
+const inverseStyle = {
```

Comment:
I'm sure there's a reason we were exporting this at one point, but there wasn't anything importing it in CK.